css/style.css

```diff
@@ -0,0 +1,143 @@
+body {
+  max-width: 100%;
+  padding: 1%;
+}
+
+.clearfix:after {
+  visibility: hidden;
+  display: block;
+  font-size: 0;
+  content: " ";
+  clear: both;
+  height: 0;
+}
+
+.hide {
+  display: none;
+}
+
+ul {
+  list-style: none;
+  padding-left: 4%;
+}
+
+h1 {
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-top: 3%;
+}
+
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+header {
+  float: left;
+  padding-bottom: 2.25%;
+  width: 100%;
+}
+
+.arrow {
+  padding-left: 4%;
+  width: 6%;
+}
+
+.listorder--float {
+  float: left;
+}
+
+.logo {
+  float: left;
+  width: 20%;
+}
+
+.main__nav {
```

Comment:
Like I mentioned earlier, It doesn't seem to me (you can always tell me I'm wrong if I'm off base here) that nav is an element within the block main. It seems like this should be `main-nav`